gh-87969: Amend docs and docstrings for ctypes.string_at and ctypes.wstring_at (#25384)
python/cpython: Misc/NEWS.d/next/Library/2021-04-13-14-58-53.bpo-43803.U_EAv6.rst

The docstrings, docs and argument lists of the functions ctypes.string_at
and ctypes.wstring_at used different names to refer to the first argument.

> Changing them to only use the name "ptr" to refer to the first argument.

Reviewer: ```suggestion
Changed them to always use the name "ptr" for the first argument.
```
